### PR TITLE
add Issue Templates and Contributing guidelines

### PR DESCRIPTION
**Pull request by thomHayner**
pushing to main because this updates how the repo functions rather than the code within that repo

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/duplicate-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/duplicate-issue.yml:1]: Should we leave this one out or no?

I would prefer to not have duplicates at all, but there are non-tech people here who might be more likely to duplicate.

Thoughts?

File: .github/ISSUE_TEMPLATE/goodfirst-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/goodfirst-issue.yml:1]: This one is already a tag.  Probably not necessary.

File: .github/ISSUE_TEMPLATE/help-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/help-issue.yml:1]: If they read the Contributing Guidelines they should be less likely to beg or ask time wasting questions, and they still might need legitimate help...feel free to object to this one being included though.

File: .github/ISSUE_TEMPLATE/invalid-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/invalid-issue.yml:1]: this is for reporting other tickets that are invalid, or other people abusing the Issues Tracker...

File: .github/ISSUE_TEMPLATE/question-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/question-issue.yml:1]: This is for people who refuse to use Telegram but have questions...

File: .github/ISSUE_TEMPLATE/wontfix-issue.yml
Comment (thomHayner) [inline .github/ISSUE_TEMPLATE/wontfix-issue.yml:1]: This one is more of an internal report for why an issue should not or won't be fixed.

**Comment by thomHayner** [comment]
@feder-cr these are basic templates for Issue Tracking.  

I started building a simple bot, it may be able to handle many of these things in the future...what do you think for now?  Should we use templates for Issues?

**Comment by feder-cr** [comment]
@thomHayner please only on V3

**Comment by feder-cr** [comment]
We will use when we do a merge with main

**Comment by feder-cr** [comment]
@thomHayner hi, any updates?

**Comment by thomHayner** [comment]
@feder-cr it should be good to go

**Pull request merged**